### ISSUE TITLE
Dynamics Randomization for MuJoCo models with nested xmls

### ISSUE THREAD

**Issue by gautams3** [rlworkgroup/garage#443]
MuJoCo files with `<include>` tags in the xml result in models being loaded from nested xmls. But when applying dynamics randomization, we can only edit dynamic parameters of the root xml. Furthermore, when we pass this edited xml to MuJoCo, we pass the entire xml as a string. This prevents MuJoCo from recognising and reading include files.

**Example:** Say we are doing variations for a Sawyer reacher task. We want to randomize the mass of the inertial node inside the[ <body> tag "right_wrist"](https://github.com/rlworkgroup/garage/blob/master/vendor/mujoco_models/sawyer_robot.xml#L65), inside sawyer_robot.xml.

We include sawyer_robot.xml through [reacher.xml](https://github.com/rlworkgroup/garage/blob/master/vendor/mujoco_models/reacher.xml#L21). However, dynamics randomization only search for tags in reacher.xml and not in the nested file sawyer_robot.xml. Even if the tag is in reacher.xml, dynamics randomization won't work because it passes the xml to MuJoCo as a string, not a path to an xml file. This fails due to improper parsing of the `<include>` tag.

**tl;dr:** Requesting these features
1. Perform search of dynamic attributes in nested xml models
2. Pass randomized xml model to MujoCo using a file path, not an xml string.
